applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -524,7 +524,7 @@ return view.extend({
 		};
 
 		o = s.taboption('advanced', form.Value, "acme_server", _("ACME server URL"),
-			_('Use a custom CA instead of Let\'s Encrypt.') +	' ' + _('Custom ACME server directory URL.'));
+			_('Use a custom CA.') +	' ' + _('Custom ACME server directory URL.'));
```

Comment:
As mentioned in the other comment, letsencrypt is indeed the default, so please keep this.